src/scaleio.rs

```diff
@@ -1531,14 +1530,20 @@ impl Scaleio {
         Ok(instances)
     }
 
-    pub fn get_drive_ids(&self) -> MetricsResult<Vec<String>> {
+    pub fn get_drive_ids(&self) -> MetricsResult<Vec<(String, String, String)>> {
```

Comment:
I'd probably make a `pub struct DriveId{ id: String, sds_id: String, storage_pool_id: String}` for this instead of the tuple or document what the tuple fields are with `///'s` because without that nobody knows what these fields are without looking up the source.

Reply:
hmm okay. most of that information is from Instance..and I could add to that for device information, alright!

Reply:
okay, done!